code/_globalvars/bitfields.dm

```diff
@@ -293,3 +293,18 @@ DEFINE_BITFIELD(zap_flags, list(
 	"ZAP_OBJ_DAMAGE" = ZAP_OBJ_DAMAGE,
 	"ZAP_GENERATES_POWER" = ZAP_GENERATES_POWER,
 ))
+
+DEFINE_BITFIELD(chemical_flags, list(
+	"REAGENT_DEAD_PROCESS" = REAGENT_DEAD_PROCESS,
+	"REAGENT_DONOTSPLIT" = REAGENT_DONOTSPLIT,
+	"REAGENT_INVISIBLE" = REAGENT_INVISIBLE,
+	"REAGENT_SNEAKYNAME" = REAGENT_SNEAKYNAME,
+	"REAGENT_SPLITRETAINVOL" = REAGENT_SPLITRETAINVOL,
+))
+
+DEFINE_BITFIELD(reaction_flags, list(
+	"REACTION_CLEAR_IMPURE"  = REACTION_CLEAR_IMPURE,
+	"REACTION_CLEAR_INVERSE" = REACTION_CLEAR_INVERSE,
+	"REACTION_CLEAR_RETAIN" = REACTION_CLEAR_RETAIN,
+	"REACTION_INSTANT" = REACTION_INSTANT
```

Comment:
```suggestion
	"REACTION_INSTANT" = REACTION_INSTANT,
```

I did this for a reason, stay consistent with the rest of the file 😄

Reply:
Didn't realise I left it out, sorry.